src/app/containers/LoginPage/index.js

```diff
@@ -48,24 +48,21 @@ function LoginPage() {
       }
       setApiProgress(API_PROGRESS.REQ);
       if (__USE_BACKUP_API__) {
-        const { apiData } = await apiLogin({ username, password });
-        console.log(apiData);
+        const { data: apiData } = await apiLogin({ username, password });
```

Comment:
@qtuan1401 api return `{ data, error }` object, so if you want to map the `data` to `apiData` you need to do this way `{ data: apiData }` then you can reference to `apiData` later than that